broker/src/test/java/io/camunda/zeebe/broker/partitioning/PartitionLeaveTest.java

```diff
@@ -57,7 +57,9 @@ void canStillProcessAfterLeaving(@TempDir final Path tmp) {
               clusterCfg.setPartitionsCount(1);
               clusterCfg.setReplicationFactor(2);
             });
-    CompletableFuture.allOf(broker0.start(), broker1.start()).join();
+    CompletableFuture.allOf(
+            CompletableFuture.runAsync(broker0::start), CompletableFuture.runAsync(broker1::start))
+        .join();
```

Comment:
:question: Do we need this test? Can this be replaced by tests in qa?

Reply:
I think this is already covered by your QA tests for scaling :+1: